### DISCUSSION TITLE
InertiaJS

### DISCUSSION THREAD

**Discussion by anjanesh** [Atri-Labs/atrilabs-engine#650] [Guides: Python Backend]
Right now, the only reason why I am learning Laravel from Django is because of InertiaJS which is easier to adapt than writing code for DRF to connect to React. With InertiaJS, the connection is auto-magical.

Wondering if the same would hold true for Atri in the near future.

With InertiaJS in Laravel I can have half the website in regular template HTML and the other half in Vue/JSX without wiring in separate backend code. It's double work without Inertia.

**Comment by cruxcode** [comment]
Hi @anjanesh, thanks for taking out time to share your experience with InertiaJS. I guess Atri framework is very much like InteriaJS. In Atri framework you can build your frontend by using visual builder, writing React code, or by writing simple HTML. The backend can communicate with the frontend seamlessly by using our simple backend Python API. This Python API is declarative in nature, hence, easy to use and you don't need to write any REST API.

**Reply by anjanesh** [reply]
This is good to know - will definitely try checking this out real soon. Will go through the docs on installation and setup.
